bolt-modules/boltlib/lib/puppet/functions/puppetize.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'fileutils'
+
+Puppet::Functions.create_function(:puppetize) do
```

Comment:
We should bikeshed over what to call this something more descriptive like apply_prep seems better to me.

My expectation is that eventually plan authors won't include this on their own it will be configured by the plan user in bolt.